roaster-zephyr/roaster-zephyr-junit5/src/main/java/hu/icellmobilsoft/roaster/zephyr/junit5/ZephyrExtension.java

```diff
@@ -113,4 +128,67 @@ private LocalDateTime getStartTime(ExtensionContext context) {
     private ExtensionContext.Store getStore(ExtensionContext context) {
         return context.getStore(ExtensionContext.Namespace.create(getClass(), context.getRequiredTestMethod()));
     }
+
+    /**
+     * Returns the number of the parameters for the test running, this number will multiply the number of the test steps,
+     * so it's default 1, it the none of the annotations exist
+     *
+     * @param context {@code ExtensionContext} context of the test execution
+     */
+    private long getTestDataListCount(ExtensionContext context) {
+        long testDataListCount = 1;
+        try {
+            for (Annotation annotation : context.getRequiredTestMethod().getDeclaredAnnotations()) {
+                if (annotation instanceof MethodSource) {
+                    testDataListCount = getTestDataListCount((MethodSource) annotation);
+                } else if (annotation instanceof EnumSource) {
+                    testDataListCount = getTestDataListCount((EnumSource) annotation);
+                } else if (annotation instanceof ArgumentsSource) {
+                    testDataListCount = getTestDataListCount((ArgumentsSource) annotation, context);
+                } else if (annotation instanceof ValueSource) {
+                    testDataListCount = getTestDataListCount((ValueSource) annotation);
+                }
```

Comment:
Ez nem az összes lehetséges source. Nem lehet a teszt lefutása után/közben elkérni a junit-tól az értéket?

Reply:
Igen, ez most az adott projekt igényeihez lett igazítva, viszont magukon a teszteken, illetve jira adminisztrációjukat is rendbe kell tenniük. Emiatt is bekerült a funkció elérhetőségét vezérlő kapcsoló, alapból kikapcsolva.